defold-spine/src/gui_node_spine.cpp

```diff
@@ -299,6 +301,12 @@ bool SetScene(dmGui::HScene scene, dmGui::HNode hnode, dmhash_t spine_scene)
     if (!resource)
         return false;
 
+    if (node->m_FindBones)
+    {
+        node->m_FindBones = 0;
+        FindBones(node);
+    }
```

Comment:
I think you need to make sure that the array fits the number of bones?
I.e. we currently only push to this array, but what if the new scene has more bones than the previous scene?

E.g.:
```
local n = gui.get_node("node_with_4_bones")
gui.set_spine_scene(n, "scene_with_16_bones")
```